### ISSUE TITLE
AGD binary error if not run from right dir

### ISSUE THREAD

**Issue by mindstyle85** [Agoric/Agoric#7825]
## Describe the bug

The new devnet binary (agoricdev-18) has an error i didnt see before, if its not being called directly from agoric-sdk/bin folder

## To Reproduce
1. Install agoricdev-18 binary

2. /home/pops/go/bin$ ./agd version
./agd: line 31: /home/pops/go/bin/../repoconfig.sh: No such file or directory

It is the same if the binary is called for example from /usr/local/bin

## Expected behavior

The compiled binary should work from any directory.

/home/pops/agoric-sdk/bin$ ./agd version
0.34.0

**Comment by ivanlei** [comment]
Does using a filesystem link help alleviate the issue?

**Comment by mindstyle85** [comment]
ill try, but i dont think that would be a solution here

**Comment by mindstyle85** [comment]
tried it, works with a link

**Comment by michaelfig** [comment]
So, just to clarify: `bin/agd` is a shell script wrapper for the actual binary.  The reason we no longer install the actual `agd` binary is that it has always relied on other parts of the Agoric SDK which we tried to make convenient to install, but which would give even harder-to-diagnose errors if installed on a machine that had references to an older SDK.

**Comment by mindstyle85** [comment]
i see, this might be then worthy to be updated in all docs etc so others are aware

**Comment by dckc** [comment]
> worthy to be updated in all docs

yes... I added a troubleshooting section to https://github.com/Agoric/agoric-sdk/releases/tag/mainnet1B-rc0 about this issue. @michaelfig I'd like to talk with you about one or two details.

**Comment by Jackwumingjie** [comment]
I have met the issue. But how to solve it?

**Comment by michaelfig** [comment]
> I have met the issue. But how to solve it?

1. If your agoric-sdk checkout is located in `$AGORIC_SDK`, then `$AGORIC_SDK/bin/agd` *must not* be copied to a different directory.
2. If you need to "install" `agd`, you must copy the entire `$AGORIC_SDK`, since `$AGORIC_SDK/bin/agd` relies on other parts of `$AGORIC_SDK` to function correctly.
3. If you want to make `agd` available somewhere else in your `$PATH`, you can use a symlink to it, like `ln -s $AGORIC_SDK/bin/agd /usr/local/bin/agd`.
